acceptance/tests/pxp_agent_version.rb

```diff
@@ -2,6 +2,9 @@
 
 test_name 'C93805 - pxp-agent - Versioning test'
 
```

Comment:
@johnduarte you can just kill this one

Reply:
@McdonaldSeanp I have added a commit to delete this test.  Thanks.